src/tools/ScenarioMeasurement/Startup/Crossgen2Parser.cs

```diff
@@ -20,29 +21,35 @@ public void EnableUserProviders(ITraceSession user)
 
         public IEnumerable<Counter> Parse(string mergeTraceFile, string processName, IList<int> pids, string commandLine)
         {
-            var loadingParser = new EventParser("Loading");
-            var emittingParser = new EventParser("Emitting");
-            var jittingParser = new EventParser("Jit");
-            var compilationParser = new EventParser("Compilation");
-
+            var loadingParser = new EventParser("Loading", (1, 2));
+            var emittingParser = new EventParser("Emitting", (3, 4));
+            var compilationParser = new EventParser("Compilation", (5, 6));
+            var jittingParser = new EventParser("Jit", (7, 8));
+            
```

Comment:
Since this is the crossgen2 parser, can we have an enum with symbolic names for these numbers? It can match whatever is used in the EventSource is in crossgen2.